### PR TITLE
[OnConnect] Cleanup

### PR DESCRIPTION
**Pull request by Arman0334**
### Description of the changes

This PR organizes the cog a bit and improves the UI of the commands.

### CODE REVIEW

File: onconnect/commands.py [hunk 1]
```diff
@@ -55,48 +59,77 @@ async def _channel(self, ctx, *, channel: Optional[discord.TextChannel] = None):
...
+                        ctx=ctx, message="Cancelled.", mention_author=True
+                    )
         else:
-            await ctx.send(
-                f"Events are already disabled.\nUse `{ctx.clean_prefix}connectset channel #channel` to enable."
+            await self.maybe_reply(
+                ctx=ctx,
+                message=(
+                    f"Events are already disabled. Use `{ctx.clean_prefix}connectset "
+                    f"channel #channel` to enable."
```
Comment (ltzmax) [inline onconnect/commands.py:116]: This is an unused f string.

File: onconnect/commands.py [hunk 2]
```diff
@@ -55,48 +59,77 @@ async def _channel(self, ctx, *, channel: Optional[discord.TextChannel] = None):
...
+
             pred = ReactionPredicate.yes_or_no(msg, ctx.author)
             try:
                 await self.bot.wait_for("reaction_add", check=pred, timeout=30)
             except asyncio.TimeoutError:
-                await ctx.send("You took too long to response, cancelling.")
+                await self.maybe_reply(
+                    ctx=ctx,
+                    message="You took too long to respond, cancelling.",
+                    mention_author=True,
```
Comment (ltzmax) [inline onconnect/commands.py:89]: I think mentions is best off by default. if people like pings, maybe a settings for it to change.

File: onconnect/commands.py [hunk 3]
```diff
@@ -55,48 +59,77 @@ async def _channel(self, ctx, *, channel: Optional[discord.TextChannel] = None):
...
+                        )
+                        await self.maybe_reply(ctx=ctx, embed=embed)
+                    else:
+                        await self.maybe_reply(
+                            ctx=ctx, message="Events have been disabled."
+                        )
                 else:
-                    await ctx.send("Cancelled.")
+                    await self.maybe_reply(
+                        ctx=ctx, message="Cancelled.", mention_author=True
```
Comment (ltzmax) [inline onconnect/commands.py:109]: Same here.

Comment (Arman0334) [reply onconnect/commands.py]: Done!

File: onconnect/commands.py (hunk 1 above)
Comment (Arman0334) [reply onconnect/commands.py]: oops, sorry about that!

File: onconnect/commands.py (hunk 2 above)
Comment (ltzmax) [reply onconnect/commands.py:89]: Alright, good point there.

**Review by ltzmax — APPROVED** [review]
LGTM thanks.

**Pull request merged**